src/components/Register.jsx

```diff
@@ -1,12 +1,42 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 
 const Register = () => {
-  const [name, setName] = useState("");
-  const [userName, setUserName] = useState("");
-  const [password, setPassword] = useState("");
+  const nagivate = useNavigate();
+  const [formInputs, setFormInputs] = useState({
+    id: new Date().getTime(),
+    name: "",
+    firstName: "",
+    password: "",
+  });
+
+  let savedProfiles = [formInputs];
+  if (localStorage.getItem("profileData")) {
+    savedProfiles = JSON.parse(localStorage.getItem("profileData"));
```

Comment:
Bhai as we are using state, no need to create a separate variable, we can use setState here